cmd/login.go

```diff
@@ -40,4 +44,5 @@ func login() {
 	if err != nil {
 		reportError("Couldn't write refresh token to config - permissions maybe?", true)
```

Comment:
"Couldn't write refresh token to config - are you able to write to `~/.config/holepunch/punch.toml`?"